ccd-definition/CaseEventToFields/CareSupervision/uploadC2.json

```diff
@@ -21,7 +21,7 @@
     "PageID": 2,
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/upload-c2/validate-pba-number/mid-event"
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/upload-c2/validate/mid-event"
   },
   {
     "LiveFrom": "01/01/2017",
```

Comment:
I have to remove this file, isn't it?

Reply:
Probably yes, or rename it to -prod? Depending how you want to tackle it. Just reviewing!

Reply:
Seems like you need to remove yes, but remember backend needs to support two ways 🤔

Reply:
So, l reintroduced `validate-pba-number/mid-event` backend code. I will test everything again but should work now.